test/journeys/specs/oneOnOne/messaging.js

```diff
@@ -114,13 +122,94 @@ describe(`Widget Space: One on One`, () => {
 
     it(`sends and deletes message`);
 
-    it(`sends message with pdf attachment`);
+    describe(`File Transfer Tests`, () => {
+      /* eslint no-sync: "off" */
+      const uploadDir = path.join(__dirname, `assets`);
+      const downloadDir = path.join(os.homedir(), `Downloads`);
+      const inputFileElem = `.ciscospark-file-input`;
+      const downloadFileButton = `(//div[@title="Download this file"]/parent::button)[last()]`;
+      const shareButtonElem = `button[aria-label="Share"]`;
+      function sendFileTest(fileName) {
+        clearEventLog(browserRemote);
+        const filePath = path.join(uploadDir, fileName);
+        const fileSize = fs.statSync(filePath).size;
+        const downloadedFile = path.join(downloadDir, fileName);
+        const fileTitle = `//div[text()="${fileName}"]`;
+        if (fs.existsSync(downloadedFile)) {
+          fs.unlinkSync(downloadedFile);
+        }
+        browserLocal.chooseFile(inputFileElem, filePath);
+        browserLocal.click(shareButtonElem);
+        browserRemote.waitForExist(fileTitle, 30000);
+        browserRemote.moveToObject(downloadFileButton);
+        browserRemote.waitForVisible(downloadFileButton);
+        const events = getEventLog(browserRemote);
+        const newMessage = events.find((event) => event.eventName === `messages:created`);
+        const fileUrl = newMessage.detail.data.files[0].url;
+        let downloadedFileSize;
+        request.get(fileUrl)
+            .set(`Authorization`, `Bearer ${mccoy.token.access_token}`)
+            .end((error, response) => {
+              downloadedFileSize = response.header[`content-length`];
+            });
+        browser.pause(8000);
```

Comment:
This will work for now, as long as we don't need more timeout for sauce tests. 

I believe you can modify the timeout for a specific test suite too, but I haven't done it yet.